### PR TITLE
v0.0.19

### PR DESCRIPTION
**Pull request by davidbrochart**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

@conda-forge-admin, please rerender

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -23,16 +23,17 @@ requirements:
     - fastapi
     - pluggy >=1.0.0,<2.0.0
     - toml
-    - click
+    - typer
```
Comment (adriendelsalle) [inline recipe/meta.yaml:26]: I thought the cli was only related to the `uvicorn` plugin, not packaged with the bare minimum `fps`. It's not the case anymore?

**Comment by adriendelsalle** [comment]
@wolfv I'm not sure bypassing the maintainer's role is the way to go on conda-forge

**Comment by davidbrochart** [comment]
> @wolfv I'm not sure bypassing the maintainer's role is the way to go on conda-forge

I also asked Wolf to merge, we know you're quite busy with other things!

File: recipe/meta.yaml (hunk 1 above)
Comment (davidbrochart) [reply recipe/meta.yaml:26]: Well, `fps` already had `click` which is related to the CLI, right?

**Comment by adriendelsalle** [comment]
@davidbrochart  IMHO it's ok if a feedstock is stuck for days/weeks/months with no answer of the original maintainer(s), which is obviously not the case here.

**Pull request merged**